libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -224,6 +224,11 @@ size_t WiFiClient::write(const uint8_t *buf, size_t size)
     return res;
 }
 
+size_t WiFiClient::write_P(PGM_P buf, size_t size)
+{
+    return write(buf, size);
+}
+
```

Comment:
PGM_P is a pointer to a string in program space. It's come from here: https://github.com/esp8266/Arduino/pull/753